common/src/main/resources/cache.properties

```diff
@@ -0,0 +1,3 @@
+tds.cache.expire.time.short=40
```

Comment:
Also adding ignoreResourceNotFound flag to CacheConfiguration so it does not break if the optional resources file is not present.